### ISSUE TITLE
Add support for JupyterLab 1

### ISSUE THREAD

**Issue by pbugnion** [pbugnion/jupyterlab-sql#84]

**Comment by stonebig** [comment]
any progress ? Jupyterlan-1.0.0 is ... today https://gitter.im/jupyterlab/jupyterlab?at=5d155dd903d62b4d2899afad

**Comment by pbugnion** [comment]
This is tracked here: #88 .

**Comment by pbugnion** [comment]
*jupyterlab-sql* 0.3.0 (now released) adds support for JupyterLab 1.0.0:

```
pip install -U jupyterlab_sql
jupyter serverextension enable jupyterlab_sql --py --sys-prefix
jupyter lab build
```
